index.html

```diff
@@ -307,6 +307,70 @@ <h3 id="type-links">External references should use typed term</h3>
           }
         </pre>
       </section>
+
+      <section>
+        <h3 id="context-order">Ordering of array elements</h3>
+        <p class="practicedesc">By default, arrays in <a class="externalDFN" href="https://www.w3.org/TR/json-ld/#sets-and-lists">JSON-LD do not convey any ordering of contained elements</a>.
+        However, for the processing of contexts, the ordering of elements in arrays <em>does</em> matter.
+        As such, when writing array-based contexts, this fact SHOULD be kept in mind.</p>
```

Comment:
I'm not sure we want to use the semi-normative `SHOULD` in this document.
```suggestion
        As such, when writing array-based contexts, this fact should be kept in mind.</p>
```